src/Services/Book/Book.Infrastructure/ServiceExtensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Book.Infrastructure;
+
+public static class ServiceExtensions
+{
+    public static void ConfigureInfrastructure(this IServiceCollection services, IConfiguration configuration)
+    {
+        
+    }
```

Comment:
why is it empty

Reply:
I implemented identity service before book service, and created these methods along with the application skeleton